.github/workflows/release.yml

```
@@ -22,7 +22,13 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ env.GO_VERSION }}
-
+      - name: Login to GitHub Packages Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

You will also need to add the `DOCKER_REGISTRY` value to the `env` block.

```yaml
env:
  GO_VERSION: "1.16"
  DOCKER_REGISTRY: "quaiy.io"
```

```suggestion
        with:
          registry: ${{ env.DOCKER_REGISTRY }}
          username: ${{ secrets.QUAY_IO_USER }}
          password: ${{ secrets.QUAY_IO_TOKEN }}
```